package.json

```diff
@@ -10,6 +10,7 @@
     "build": "tsc -p .",
     "replaceVars": "ts-node --transpile-only src/replace.ts bin/analytics.js",
     "test": "echo \"Error: no test specified\" && exit 1",
+    "typecheck": "tsc -p . --noEmit",
```

Comment:
Should this be included in the `lint` script below? I figured type-checking is a part of linting.

Reply:
Makes sense

Reply:
I added it separately because to properly use eslint and typecheck for `lint-staged` plugin it needed to be separate